README.md

```diff
@@ -1,41 +1,30 @@
-Puppet & Datadog
-================
+# Datadog Puppet Module
 
-[![Build Status](https://img.shields.io/circleci/build/gh/DataDog/puppet-datadog-agent.svg)](https://circleci.com/gh/DataDog/puppet-datadog-agent)
-[![Puppet Forge](https://img.shields.io/puppetforge/v/datadog/datadog_agent.svg)](https://forge.puppetlabs.com/datadog/datadog_agent)
-[![Puppet Forge Downloads](https://img.shields.io/puppetforge/dt/datadog/datadog_agent.svg)](https://forge.puppetlabs.com/datadog/datadog_agent)
```

Comment:
I like having these on our open source repos. Is there any reason not to have it?

Reply:
Oh, this is probably because we pull this README into our docs, am I right? It would be nice if there was a way to keep these here without them appearing in the docs... but if that's hard to do we can go without them.

Reply:
Yep. I moved them to the contributing file.